web/static/web/css/header.css

```diff
@@ -1,144 +1,332 @@
+.lang-link .item .text {
+    color: #DCDDDE;
 }
 
-.nav .logo {
+#header .nav-item:hover .nav-item-text,
+#nav-user-dropdown .dropdown-item:hover .dropdown-item-text,
+.lang-link .item:hover .text {
+    color: #FFF;
+}
+
+#header div.logo-centering {
+    display: none;
+}
+
+#header .logo {
+    display: grid;
+    grid-template-columns: var(--logo-padding) var(--logo-space) minmax(var(--nav-space-between), var(--logo-padding));
     align-self: center;
-    padding: 0 30px;
 }
 
-.nav .logo img {
+#header .logo img {
+    margin: auto;
+    max-height: 50px;
     object-fit: contain;
 }
 
-.limg {
+#header .logo .limg.large {
+    display: block;
+}
+
+#header .logo .limg.medium {
     display: none;
 }
 
-.limg.large {
-    display: inline;
+#header .logo .limg.small {
+    display: none;
 }
 
-.burger {
+#header #burger {
     display: none;
+    align-self: center;
+    justify-self: end;
+    margin-right: var(--burger-margin);
     margin-bottom: -10px;
 }
 
-.burger:hover {
+#header #burger:hover {
     cursor: pointer;
 }
 
-.burger {
-    align-self: center;
+#header #burger i {
+    color: #FFFFFF;
+    font-size: 30px;
+}
+
+#header #nav {
     justify-self: center;
+    font-size: var(--font-size);
 }
 
-.burger i {
-    color: #FFFFFF;
-    font-size: 30px;
+#header #nav .nav-item,
+#header #side-nav .nav-item {
+    height: calc(var(--font-size) + 2em);
+    margin: calc(var(--nav-space-between) / 2);
+    padding: 0;
+    border-radius: 0;
+    justify-content: center;
 }
 
-.nav .ui.inverted.menu .item {
-    color: rgba(255, 255, 255, .7) !important;
+#header #nav .nav-item:hover,
+#header #side-nav .nav-item:hover,
+#nav-user-dropdown.active {
+    /* Disables property from Semantic UI */
+    background-color: transparent;
 }
 
-.lang-form {
-    display: none;
+#header #nav .nav-item .bubble-background,
+#header #side-nav .nav-item .bubble-background {
+    position: absolute;
+    width: 100%;
+    height: var(--font-size);
+    border-radius: 2em;
+    z-index: 1;
+    transition: width var(--dur) linear, height var(--dur) linear, background-color var(--dur) ease-out;
+}
+
+#header #nav .nav-item:hover .bubble-background,
+#header #side-nav .nav-item:hover .bubble-background {
+    width: calc(100% + var(--bubble-background-extra-width));
+    height: calc(var(--font-size) + 2em);
+    z-index: 2;
+}
+
+#header #nav .nav-item .nav-item-text,
+#header #side-nav .nav-item .nav-item-text {
+    z-index: 3;
+    transition: color var(--dur) ease;
+}
+
+#header #nav .nav-item:nth-child(1):hover .bubble-background {
+    background-color: rgba(239, 184, 14, 0.9);
+}
+
+#header #nav .nav-item:nth-child(2):hover .bubble-background {
+    background-color: rgba(0, 235, 179, 0.8);
+}
+
+#header #nav .nav-item:nth-child(3):hover .bubble-background {
+    background-color: rgba(12, 100, 141, 1); /* MAKE's blue primary color (12, 66, 106), with brightness turned up from 149 to 114 */
```

Comment:
Because it was too dark on its own, relative to the background and the other colors. Like the comment says, it's the same color as in the graphic profile manual, just with brightness turned up (which technically doesn't make it the same color, I know :p ), and I think that it still looks like it fits, which is the most important thing, I think.

Reply:
It is probably fine to use here.